OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -118,7 +118,7 @@ public void UpdateMouseover()
 			}
 
 			var frozen = world.ScreenMap.FrozenActorsAt(world.RenderPlayer, worldRenderer.Viewport.ViewToWorldPx(Viewport.LastMousePos))
-				.Where(a => a.TooltipInfo != null && a.IsValid)
+				.Where(a => a.TooltipInfo != null && a.IsValid && (a.Actor != null && !a.Actor.IsInvisibleTo(world.RenderPlayer)))
```

Comment:
Here's my understanding (correct me if I've got this wrong as I'm somewhat confused right now):
- We want to run ```!a.Actor.IsInvisibleTo(world.RenderPlayer)``` to stop the tooltip appearing.
- We don't want to do ```a.Actor != null``` as that would introduce an issue if the frozen actor was dead.
- But we can't call ```IsInvisibleTo``` without checking if the actor is null first, so we can't satisfy both of the above requirements?